hw/ip/otbn/dv/uvm/get-toolchain-paths.sh

```diff
@@ -13,18 +13,18 @@ if [[ -n ${BAZEL_CACHE} ]]; then
   ${BAZEL_CMD} fetch \
     --distdir="${BAZEL_DISTDIR}" \
     --repository_cache="${BAZEL_CACHE}" \
-    @lowrisc_rv32imcb_files//...
+    @lowrisc_rv32imcb_$(uname -m)_files//...
```

Comment:
Would it be better to capture the output of `uname` into a variable (e.g. `HOST_ARCH` or similar) and then expanding that variable where needed?

We could use the bash `:=` assignment to set the variable if not already set, which would permit overriding the value from the external environment.
```
: ${HOST_ARCH:=$(uname -m)}
```
(see https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html)